.github/workflows/azure-e2e-test.yml

```diff
@@ -20,6 +20,12 @@ on:
       caa-image:
         type: string
         description: prebuilt caa image
+      kbs-image-id:
+        type: string
+        description: prebuild kbs image
+      kbs-image-tag:
+        type: string
+        description: prebuild kbs image tag
```

Comment:
Can't we read all this information from `versions.yaml` ?

Reply:
Yes we are reading it from version.yaml and we are also providing an option to give it in CI.